api/alert_channels_webhook_test.go

```diff
@@ -0,0 +1,135 @@
+	"github.com/lacework/go-sdk/internal/intgguid"
+	"github.com/lacework/go-sdk/internal/lacework"
+)
+
+func TestAlertChannelsGetWebhook(t *testing.T) {
+	var (
+		intgGUID   = intgguid.New()
+		apiPath    = fmt.Sprintf("AlertChannels/%s", intgGUID)
+		fakeServer = lacework.MockServer()
+	)
+	fakeServer.UseApiV2()
+	fakeServer.MockToken("TOKEN")
+	defer fakeServer.Close()
+
+	fakeServer.MockAPI(apiPath, func(w http.ResponseWriter, r *http.Request) {
+		assert.Equal(t, "GET", r.Method, "GetWebhook() should be a GET method")
+		fmt.Fprintf(w, generateAlertChannelResponse(singleWebhookAlertChannel(intgGUID)))
+	})
+
+	c, err := api.NewClient("test",
+		api.WithApiV2(),
+		api.WithToken("TOKEN"),
+		api.WithURL(fakeServer.URL()),
+	)
+	assert.Nil(t, err)
+
+	response, err := c.V2.AlertChannels.GetWebhook(intgGUID)
+	assert.Nil(t, err)
+	assert.NotNil(t, response)
+	assert.Equal(t, intgGUID, response.Data.IntgGuid)
+	assert.Equal(t, "integration_name", response.Data.Name)
+	assert.True(t, response.Data.State.Ok)
+	assert.Contains(t, response.Data.Data.Url, "https://hooks.webhook.com/?api-token=12345")
+}
+
+func TestAlertChannelWebhookUpdate(t *testing.T) {
+	var (
+		intgGUID   = intgguid.New()
+		apiPath    = fmt.Sprintf("AlertChannels/%s", intgGUID)
+		fakeServer = lacework.MockServer()
+	)
+	fakeServer.UseApiV2()
+	fakeServer.MockToken("TOKEN")
+	defer fakeServer.Close()
+
+	fakeServer.MockAPI(apiPath, func(w http.ResponseWriter, r *http.Request) {
+		assert.Equal(t, "PATCH", r.Method, "UpdateWebhook() should be a PATCH method")
+
+		if assert.NotNil(t, r.Body) {
+			body := httpBodySniffer(r)
+			assert.Contains(t, body, intgGUID, "INTG_GUID missing")
+			assert.Contains(t, body, "integration_name", "alert channel name is missing")
+			assert.Contains(t, body, "Webhook", "wrong alert channel type")
+			assert.Contains(t, body, "https://hooks.webhook.com/?api-token=12345", "missing webhook url")
+		}
+
+		fmt.Fprintf(w, generateAlertChannelResponse(singleWebhookAlertChannel(intgGUID)))
+	})
+
+	c, err := api.NewClient("test",
+		api.WithApiV2(),
+		api.WithToken("TOKEN"),
+		api.WithURL(fakeServer.URL()),
+	)
+	assert.Nil(t, err)
+
+	webhookAlertChan := api.NewAlertChannel("integration_name",
+		api.WebhookAlertChannelType,
+		api.WebhookDataV2{
+			Url: "https://hooks.webhook.com/?api-token=12345",
+		},
+	)
+	assert.Equal(t, "integration_name", webhookAlertChan.Name, "Webhook alert channel name mismatch")
+	assert.Equal(t, "Webhook", webhookAlertChan.Type, "a new Webhook alert channel should match its type")
+	assert.Equal(t, 1, webhookAlertChan.Enabled, "a new Webhook alert channel should be enabled")
+	webhookAlertChan.IntgGuid = intgGUID
+
+	response, err := c.V2.AlertChannels.UpdateWebhook(webhookAlertChan)
+	assert.Nil(t, err)
+	assert.NotNil(t, response)
+	assert.Equal(t, intgGUID, response.Data.IntgGuid)
+	assert.True(t, response.Data.State.Ok)
+	assert.Contains(t, response.Data.Data.Url, "https://hooks.webhook.com/?api-token=12345")
+}
```

Comment:
nit: Can use fmt.Sprintf here instead of string concatenation

Reply:
There's no concatenated string in this segment of code.